vmm/src/cpu.rs

```diff
@@ -811,6 +811,8 @@ impl CpuManager {
     ) -> Result<()> {
         let reset_evt = self.reset_evt.try_clone().unwrap();
         let exit_evt = self.exit_evt.try_clone().unwrap();
+        #[cfg(feature = "kvm")]
+        let on_kvm: bool = HypervisorType::Kvm == self.hypervisor_type;
```

Comment:
Nit: it would be cleaner if you do:
```suggestion
        #[cfg(feature = "kvm")]
        let hypervisor_type = self.hypervisor_type;
```